### ISSUE TITLE
Please update auto-converted files, I need .opi

### ISSUE THREAD

**Issue by oksanagit** [epics-modules/xspress3#16]
I was having a list of disconnected PVs on .opi screens which I took from the project. After looking around I found a tiny discrepancy in names provide by epics IOC and used in .opi screens. Then I looked at the project updates. It seems adl folder has been updated 3 month ago and the rest including opi had been  8 month ago. I concluded that perhaps a conversion could fix the situation.
Thank you,

**Comment by MarkRivers** [comment]
I have done a git pull to get the latest master branch, and then done the autoconvert of OPI screens.  I then pushed the new and modified files back to Github.

**Comment by oksanagit** [comment]
Thank you @MarkRivers . It didn't seem to fix the situation. I use ADCore 3.8. Most of the errors are related to unconnected Time Series PVs on the screens provided by areaDetector.

![xspress3_0](https://user-images.githubusercontent.com/21367956/91361549-90606a00-e7c6-11ea-9c38-a170b15144cc.JPG)

xspress3_0: the ROIs are just not configured and could be ignored. The PVs ``` $P$R:C1SCA:0:Value_RBV```   ```$P$R:C1SCA:1:Value_RBV``` .... ``` $P$R:C1SCA:10:Value_RBV```   are not provided by IOC I have. Instead I see ```$P$R:C[1-4]SCA[1-8]:Value_RBV ```
![xspress3_1](https://user-images.githubusercontent.com/21367956/91362346-4b3d3780-e7c8-11ea-94ae-b5d96726de2a.JPG)
![xspress3_2](https://user-images.githubusercontent.com/21367956/91362526-ae2ece80-e7c8-11ea-8ad7-bed8f419edf2.JPG)

I checked, and I don't see any database errors in my start log.  Maybe I am not supposed to have these PVs available?
Thank you.

**Comment by MarkRivers** [comment]
I think the problem is probably some macro to set the PV name.

At the IOC prompt type
```
dbgrep *TSNumPoints
```
That will show the names of the records that end in TSNumPoints.  Compare those to the names that CSStudio is looking for.  There is probably some macro that is incorrect either when loading the database or invoking the CSS screen.

**Comment by MarkRivers** [comment]
@oksanagit did you have a chance to look at this?

**Comment by oksanagit** [comment]
Yes, I did. The reason for me having disconnected PVs was because the ...iocs/iocxspress3-4Channel/SCARIO.cmd  did not load a list of plugins which provide those.  ...iocs/iocGSECARD-4Channel/SCARIO.cmd does.  When I used iocGSECARD-4Channel/ for my IOC, everything seem to connect.  It seems both IOC directories provide IOC for the same type of hardware. 

Though, our users (knowledgeable users, who worked with xspresses)  said they do not use scanning and other missing functionality, having pink disconnected PVs  makes bad impression in any case. 

I think this issue can be closed, screens are updated and they work, they look nice with all text displayed. 
I was wondering if it could make sense to remove iocxspress3-4Channel at all, since it is inferior to /iocGSECARD-4Channel/ and both support the same hardware. I somehow think that people would be happy to start with GSECARD configuration. I did.
Thank you,

Oksana
